src/internal/deploy/cmds/cmds.go

```diff
@@ -537,7 +546,19 @@ func standardDeployCmds() []*cobra.Command {
 				// Serve the Pachyderm object/block API locally, as this is needed by
 				// our tests (and authentication is disabled anyway)
 				opts.ExposeObjectAPI = true
+
+				// Set the postgres and etcd nodeports explicitly for developers
+				opts.PostgresOpts.Port = 32228
+				opts.EtcdOpts.Port = 32379
 			}
+
+			// Put the enterprise server backing data in a different path,
```

Comment:
What sort of data goes here? Is this just forward looking at the moment? I thought the state was in postgres?